build.gradle

```diff
@@ -8,10 +8,9 @@ buildscript {
     }
 
     dependencies {
-        classpath("com.android.tools.build:gradle:4.1.2")
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version")
-        classpath("com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3")
-        classpath("com.github.dcendents:android-maven-gradle-plugin:2.0")
+        classpath "com.android.tools.build:gradle:4.1.2"
```

Comment:
update to 4.1.3 pls

Reply:
d956d576a63e6440578d5cab79a663f352f02ccc